dev/podman/setup-local-env-podman.md

```diff
@@ -0,0 +1,53 @@
+## The following are Podman specific steps used to set up on a MacBook (Intel or Apple Silicon)
+
+### Verify installed tools (install if needed)
+
+```sh
+podman --version
+podman version 5.0.1
+kind version
+kind v0.23.0 go1.22.3 darwin/arm64
+tilt version
+v0.33.12, built 2024-03-28
+```
+
+### Start Kind with a local registry
+Use this [helper script](./kind-with-registry-podman.sh) to create a local single-node Kind cluster with an attached local image registry.
+
+You'll need to disable secure access on the local kind registry:
+
+`podman inspect kind-registry --format '{{.NetworkSettings.Ports}}'`
+With the port you find for 127.0.0.1 edit the config file:
```

Comment:
This seems incomplete.

Reply:
Is this supposed to be referencing the next section? In that case I'd suggest using 4 hashes for the section header below.

Reply:
This sections is off... I'll rephrase it to make sense.